surefire-providers/surefire-testng/src/main/java/org/apache/maven/surefire/testng/conf/TestNG740Configurator.java

```diff
@@ -37,33 +35,45 @@
  *
  * @since 3.0.0-M6
  */
-public class TestNG740Configurator extends TestNG60Configurator
+public class TestNG740Configurator
+    extends TestNG60Configurator
```

Comment:
Nit: can you put the extends back on the same line as the class.

Reply:
This is the style employed by **all** of the other classes in this project.

Reply:
guys, it's better to avoid doing any refactoring. It's only harder for reviewers.

Reply:
This specific change was a minor format change to match the style used throughout this project.